apps/console/src/features/applications/components/settings/attribute-management/advance-attribute-settings.tsx

```diff
@@ -133,6 +140,12 @@ export const AdvanceAttributeSettings: FunctionComponent<AdvanceAttributeSetting
         }
     }, [ dropDownOptions ]);
 
+    useEffect( () => {
```

Comment:
```suggestion
    useEffect(() => {
```
Consider defining a dependency array to avoid unnecessary executions.

Reply:
fixed with [be309bd](https://github.com/wso2/identity-apps/pull/4284/commits/be309bdd61f1ecdcf606cdf0bfe85ea775a210a6)